11ty/definitions/guys-gals.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Guys and Gals
```

Comment:
Can we please break these into two separate word entries? For clarity, I will review this as though it's for "guys"